.github/workflows/pipeline.yml

```
@@ -48,7 +49,7 @@ jobs:
     uses: ./.github/workflows/gradle_localstack_postgres_verify.yml # WORKFLOW_VERSION
     secrets: inherit
     with:
-      java-options: "${{ inputs.java_options }}"
+      gradle-command: "make test-all"
```

```suggestion
      gradle-command: "export _JAVA_OPTIONS=${JAVA_OPTS} && make test-all"
```
This means we only have to specify the `JAVA_OPTIONS` in the pipeline and will make it easier for people to reuse if we contribute it back. The PR is fine as is, can we do anything to make it more reusable?

---

Yeah might be worth updating the shared action to do this, gonna update to have a reusable java opts